core/src/main/resources/reference.conf

```diff
@@ -29,4 +29,9 @@ health-monitor-dispatcher {
 
 gcs {
   bucketLogsMaxAge = "180"
+}
+
+submissionmonitor {
+  submissionPollInterval = 1m
```

Comment:
Just curious - why did you set this submissionPollInterval to 1m? Is it arbitrary? Or is there a reasoning behind it?